### ISSUE TITLE
Plugin ignores `.webm` files

### ISSUE THREAD

**Issue by kdheepak** [bluwy/svelte-preprocess-import-assets#5]
Thanks again for the plugin. 

I'm not able to process `.webm` files with this plugin. Is that something I have to handle manually or is there something I can do to get that to work here? Or maybe the plugin can be updated to handle this case? 

I have something like this in markdown:

`![filename](videos/filename.webm)`

and the file does not show up in the assets folder when I try to build the project.

**Comment by bluwy** [comment]
The preprocessor doesn't check for extensions when transforming the markup. So if you have:

```html
<img src="videos/filename.webm" alt="filename" />
```

It should transform it into an import like `import ___ASSET___ from "videos/filename.webm"`. Perhaps you need a leading `./` to fix it? But it also doesn't look like videos should be in an `img` tag.

**Comment by kdheepak** [comment]
I did a little more digging and my source gets transformed into this:

```
<video src="./videos/filename.webm" loop autoplay style="width:100.0%" controls><a href="./videos/filename.webm">
```

But this is not converted into a asset. Any idea what might be going on?

**Comment by kdheepak** [comment]
I submitted a PR. I made this change locally in the node_modules dist folder and it appears to work.

**Comment by bluwy** [comment]
This is fixed in 0.1.5. Thanks!

**Comment by kdheepak** [comment]
Thanks!
